src/components/m-button/index.js

```diff
@@ -6,8 +8,23 @@ import Button from './js/button';
 
 class AXAButton extends BaseComponentGlobal {
   static tagName = 'axa-button'
+  static propTypes = {
+    arrow: PropTypes.bool,
+    classes: PropTypes.string,
+    color: PropTypes.oneOf(['red', 'white']),
+    ghost: PropTypes.bool,
+    motion: PropTypes.bool,
+    size: PropTypes.oneOf(['sm', 'lg']),
```

Comment:
Should we include `md`?
Medium size is default and hence has no extra classes, though it's not wrong 🤔

Reply:
yes better include it